components/eam/src/physics/cam/shoc.F90

```diff
@@ -192,7 +192,11 @@ subroutine shoc_main ( &
      w_sec, thl_sec, qw_sec, qwthl_sec,&  ! Output (diagnostic)
      wthl_sec, wqw_sec, wtke_sec,&        ! Output (diagnostic)
      uw_sec, vw_sec, w3,&                 ! Output (diagnostic)
-     wqls_sec, brunt, shoc_ql2)           ! Output (diagnostic)
+     wqls_sec, brunt, shoc_ql2,&          ! Output (diagnostic)
```

Comment:
Does the comma at the end of this line mess things up when `SCREAM_CONFIG_IS_CMAKE` isn't defined?